shared/test/test_assets.rb

```diff
@@ -3,6 +3,7 @@
 require File.expand_path '../../../deployment', __FILE__
 require File.expand_path '../../middleware/files_api', __FILE__
 require File.expand_path '../../middleware/channels_api', __FILE__
+require File.expand_path '../../middleware/helpers/asset_bucket', __FILE__
```

Comment:
Thinking about this more, we could change the FilesApi calls in test_assets_copy_all to hit AssetBucket directly. That would make this a cleaner unit test, but it would make the test more different from the access pattern we expect in production, so it seems like a net loss.